src/components/List/List.tsx

```diff
@@ -45,6 +46,7 @@ export interface ListProps<T> extends Omit<ListGroupProps, 'onSelect'> {
   sortByLabel?: SortHeaderProps['sortByLabel'];
   sortOptions?: SortHeaderProps['sortOptions'];
   selectable?: (item: T) => boolean;
+  selectAnywhere?: boolean;
```

Comment:
Would it be valuable to add an entry to `defaultProps` ?

```
const defaultProps = {
  ...,
  selectAnywhere: false
};
```

Reply:
I didn't find that this was necessary (I think omitted props default to false anyways?)

Reply:
I think they default to `undefined`, but this would help document that the datatype of `selectAnywhere` is a boolean.